docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -6,7 +6,9 @@ You will also deploy a simple application that uses that gateway for ingress and
 We will start with a single cluster and move to multiple clusters to illustrate how a single gateway definition can be used across multiple clusters and highlight the automatic TLS integration and also the automatic DNS load balancing between gateway instances.
 
 ## Requirements
-- [Kind](https://kind.sigs.k8s.io/)
+- [kind](https://kind.sigs.k8s.io/)
+- [operator-sdk](https://sdk.operatorframework.io/docs/installation/)
+- [yq](https://mikefarah.gitbook.io/yq/v/v3.x/)
```

Comment:
The script doesn't appear to install dependencies right? Does that mean you need to list all of them here? ocm clusteradm etc...

Reply:
In the current state it doesn't install dependencies no. The docs edit here is also just a suggestion. If we'd prefer to stick with make local-setup for this one it can be reverted. If we're happy to go with the change I'll get clusteradm added in as well. Missed that one. Good spot.